src/commands/debug.ts

```diff
@@ -0,0 +1,26 @@
+import type { GuildMember } from "discord.js";
+import type { Command } from "../command";
+import { log, logLevels } from "../logger";
+
+const command: Command = {
+	name: "debug",
+	description: "Test logger.",
+	options: [],
+	hasPermission: (state, interaction) =>
+		(interaction.member as GuildMember).roles.cache.has(state.config.developerRoleId),
+	shouldBeEphemeral: (state, interaction) => interaction.channelID !== state.config.botChannelId,
+	handler: (state, interaction) => {
+		interaction
+			.reply("This command helps with intentional debug testing.")
```

Comment:
```suggestion
			.reply("This command helps with intentional debug testing.", { ephemeral: command.shouldBeEphemeral(state, interaction) })
```

Perhaps make this ephemeral? Maybe not, as it is dev only anyways.